Utilities/ReleaseScripts/scripts/git-publish

```diff
@@ -405,9 +405,8 @@ CMSSW_CATEGORIES={"Operations": ["Configuration/Applications",
   "AnalysisDataFormats/SUSYBSMObjects", "AnalysisDataFormats/SiStripClusterInfo",
   "AnalysisDataFormats/TopObjects", "AnalysisDataFormats/TrackInfo",
   "DiffractiveForwardAnalysis/Configuration",
-  "DiffractiveForwardAnalysis/Skimming", "EgammaAnalysis/CSA07Skims",
-  "EgammaAnalysis/Configuration", "EgammaAnalysis/ElectronIDESSources",
-  "EgammaAnalysis/PhotonIDProducers", "ElectroWeakAnalysis/Configuration",
+  "DiffractiveForwardAnalysis/Skimming",
+  "ElectroWeakAnalysis/Configuration",
```

Comment:
I seem to recall that this file is supposed to contain all known packages from the past, not just the current release.

Reply:
Really? I did the same thing in another PR where I removed a package:
https://github.com/cms-sw/cmssw/pull/26186
Back then, @Dr15Jones was giving a +1 for that. So maybe I should also revert that change?

Reply:
nevermind, I may have misunderstood/confused with another file with a list of packages.
 This one is in the core signautre list anyways and will be confirmed.